index.html

```diff
@@ -3614,6 +3640,31 @@ <h3>Fingerprinting Network Requests</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Canonicalization Method Privacy</h3>
+
+        <p>
+The way in which transformation, namely canonicalization, is performed can
+affect the privacy characteristics of a system. Selecting among these
+canonicalization mechanisms is dependent on the use case being addressed.
+This section attempts to provide simple guidance to help
+implementers pick between the two main canonicalization mechanisms referred to
+in this specification, namely JSON Canonicalization Scheme [[RFC8785]] and
+RDF Dataset Canonicalization [[RDF-CANON]], from a privacy perspective.
+        </p>
+        <p>
+If an application does not require performing a selective disclosure of
+information in a secured document, nor does it utilize JSON-LD, then JSON
+Canonicalization Scheme [[RFC8785]] is most likely the best approach.
+        </p>
+        <p>
+If an application utilizes JSON-LD or might require the selective disclosure
+of information in a secured document, then using a cryptography suite that
```

Comment:
This is probably my missing domain knowledge, but I do not understand the second half of the sentence ("or might require..."). Can you explain?

Reply:
Yeah something grammatically doesn't seem correct here and I'm not picking up on what this is trying to say. Would be good to rephrase it I'm thinking.

Reply:
Hmm, it made sense to me. It reads as: "If an application ... might require selective disclosure of information in a secured document" .

Reply:
Ahh on re-reading this I figured out what was confusing for me. It would make sense if it's an AND because data-integrity is going to utilize JSON-LD and with SD then RDF-canon makes sense. If people just want SD without semantic security it's probably simpler to just point them to JWT-SD. Would we really want to point people at data integrity suites if they don't care about JSON-LD and want selective disclosure? Seems like forcing them to take on RDF-Canon if they only want SD without caring about the semantics is a bit more complex than necessary.

Reply:
@kdenhartog I've added some language to point people to securing mechanisms that don't do canonicalization in this commit: 464ac07744ae57de118fdf7f5813215b3a6e0035

Reply:
@iherman and @kdenhartog I addressed the "and/or" issue via this commit: c3ffc667d906b51766e7763d4f977a7c8dbc5d02